### DISCUSSION TITLE
Is there any Toggle button without group?

### DISCUSSION THREAD

**Discussion by nizarbathery** [quasarframework/quasar#14017] [General - Components / Directives / etc]
Hi
I am new to Quasar framework, I was trying to use a card or button behave like toggle button, but from my understanding the toggle button is only work in group. I actually wanted to display group of standard buttons and change the color of clicked one. I wanted to display in a table grid and user can click one (single click mode). I tried Toggle but I cannot hide the switch (just need the functionality behind and change the background color (selected). Any tips would be great.  

I am looking for something like in bootstrap button below
```
<button type="button" class="btn btn-primary" data-toggle="button" aria-pressed="false" autocomplete="off">
  Single toggle
</button>
```

**Comment by MilosPaunovic** [comment]
You could use regular button and toggle value on click event like this: `@click="value = !value`.
